src/mame/mr/nl_dribling.cpp

```diff
@@ -0,0 +1,494 @@
+	RES(R35, RES_K(1))
+	RES(R36, RES_K(56))
+	RES(R37, RES_K(10))
+	RES(R47, RES_K(330))
+	CAP(C20, CAP_N(100))
+	CAP(C21, CAP_N(10))	// 10K?
+	CAP(C22, CAP_N(22))	// 22K?
+	//CAP(C23, CAP_U(0.1))
+	CAP(C24, CAP_U(10))
+
+	NET_C(IC_8B.1, IC_7B.4)
+	NET_C(IC_8B.2, IC_8A.3)
+	NET_C(IC_8B.3, IC_8A.1, IC_8A.2)
+	NET_C(IC_8A.9, IC_7A.9, I_V5)
+	NET_C(IC_8A.8, IC_7A.8, R32.1, IC_7B.6)
+	NET_C(IC_8A.13, IC_7A.1, IC_7A.2)
+	NET_C(IC_7A.12, IC_7B.3)
+	NET_C(R32.2, IC_7B.5, C20.2)
+	NET_C(C20.1, GND)
+	ALIAS(NOISE, IC_7A.13)
+	ALIAS(C_IN, IC_8A.8)
+
+	NET_C(I_FOLLA_B, IC_E8.10)
+	NET_C(I_FOLLA_M, IC_E8.12)
+	NET_C(I_FOLLA_A, IC_E8.1)
+	NET_C(IC_E8.9, IC_E8.13, IC_E8.2, NOISE)
+	NET_C(IC_E8.8, R33.1)
+	NET_C(IC_E8.11, R34.1)
+	NET_C(IC_E8.3, R36.1)
+	NET_C(R35.1, GND)
+	NET_C(R33.2, R34.2, R35.2, R36.2, C22.1, C21.1)
+	NET_C(C21.2, R47.2, IC_C8.14)
+	ALIAS(FOLLA, IC_C8.14)
+	NET_C(R37.2, I_V5)
+	NET_C(C22.2, R47.1, IC_C8.13)
+	NET_C(R37.1, C24.2, IC_C8.12)
+	NET_C(C24.1, GND)
+
+    // CONTRASTO
+	SUBMODEL(NE556_DIP, IC_Q9)		// VCO
+	NET_C(IC_Q9.7, GND)
+	NET_C(IC_Q9.14, I_V5)
+
+    RES(R38, RES_M(1))
+	RES(R39, RES_K(100))
+	CAP(C23, CAP_U(0.1))
+	CAP(C25, CAP_U(0.1))
+	CAP(C26, CAP_U(0.47))
+	NET_C(R38.1, I_V5)
+	NET_C(R39.1, I_V5)
+	NET_C(R38.2, IC_Q9.1, IC_Q9.2, C25.1)
+	NET_C(C25.2, C26.2, GND)
+	NET_C(R39.2, IC_Q9.13, IC_Q9.12, IC_Q9.8, C26.1)
+	NET_C(IC_Q9.5, IC_Q9.10)
+	NET_C(IC_Q9.9, C23.1)
+	NET_C(IC_Q9.4, I_V5) // ??
+	NET_C(I_CONTRASTO, IC_Q9.6)
+	ALIAS(CONT_OUT, C23.2)
+	NET_C(C23.2, R36.2)
+
+	// OUTPUT SECTION ----------------------
+	// TOS
+	CAP(C_TOS, CAP_N(100))
+	RES(R_TOS, RES_K(220))
+	NET_C(TOS, C_TOS.1)
+	NET_C(R_TOS.1, C_TOS.2)
+
+	// FISCHIO
+	CAP(C_FISCHIO, CAP_N(1000))
+	RES(R_FISCHIO, RES_K(220))
+	NET_C(FISCHIO, C_FISCHIO.1)
+	NET_C(R_FISCHIO.1, C_FISCHIO.2)
+
+	// FOLLA
+	CAP(C_FOLLA, CAP_N(10))
+	RES(R_FOLLA, RES_K(100))
+	NET_C(FOLLA, C_FOLLA.1)
+	NET_C(C_FOLLA.2, R_FOLLA.1)
+
+	// CALCIO_A 
+	CAP(C_CALCIO_A, CAP_U(1))
+	RES(R_CALCIO_A, RES_K(10))
+	NET_C(CALCIO_A.OUTPUT, C_CALCIO_A.1)
+	NET_C(C_CALCIO_A.2, R_CALCIO_A.1)
+
+	// CALCIO_B
+	CAP(C_CALCIO_B, CAP_U(1))
+	RES(R_CALCIO_B, RES_K(10))
+	NET_C(CALCIO_B.OUTPUT, C_CALCIO_B.1)
+	NET_C(C_CALCIO_B.2, R_CALCIO_B.1)
+
+	// STOP_PALLA
+	CAP(C_STOP_PALLA, CAP_U(1))
+	RES(R_STOP_PALLA, RES_K(100))
+	NET_C(STOP_PALLA.OUTPUT, C_STOP_PALLA.1)
+	NET_C(C_STOP_PALLA.2, R_STOP_PALLA.1)
+
+	// PARATA
+	CAP(C_PARATA, CAP_U(1))
+	RES(R_PARATA, RES_K(100))
+	NET_C(PARATA.OUTPUT, C_PARATA.1)
+	NET_C(C_PARATA.2, R_PARATA.1)
+
+	// FINAL MIX AND DISABLE CIRCUITRY
+	TTL_INPUT(ENABLE_SOUND, 0)
+	NET_C(ENABLE_SOUND.VCC, I_V5)
+	NET_C(ENABLE_SOUND.GND, GND)
+	QBJT_EB(Q_OUT, "BC239C")
+	RES(R_OUT, RES_K(2.2))
+	RES(R_PU_OUT, RES_K(10))
+	NET_C(R_PU_OUT.1, I_V5)
+	NET_C(ENABLE_SOUND, R_OUT.1, R_PU_OUT.2)
+	NET_C(Q_OUT.B, R_OUT.2)
+	NET_C(Q_OUT.C, R_TOS.2, R_FISCHIO.2, R_FOLLA.2, R_CALCIO_A.2, R_CALCIO_B.2, R_STOP_PALLA.2, R_PARATA.2)
+	NET_C(Q_OUT.E, GND)
+
+	ALIAS(OUTPUT, Q_OUT.C)
+}
```

Comment:
This file is missing the ending newline.

Reply:
done